### PR TITLE
[luci] Support FuseBCQPass for Module

### PR DESCRIPTION
**Pull request by llFreetimell**
Parent Issue : #4961

Until now, `FuseBCQPass` was just available for `loco::Graph`.
This commit will enable supporting for `luci::Module` and change
`run(loco::Graph *)` to do nothing.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by mhs4670go — DISMISSED** [review]
LGTM

File: compiler/luci/pass/include/luci/Pass/FuseBCQPass.h [hunk 1]
```diff
@@ -26,11 +26,12 @@ namespace luci
...
  *
  */
-struct FuseBCQPass final : public logo::Pass
+struct FuseBCQPass final : public luci::Pass
 {
   const char *name(void) const final { return "luci::FuseBCQPass"; }
 
-  bool run(loco::Graph *g) final;
+  bool run(luci::Module *m) final;
+  bool run(loco::Graph *) final;
```
Comment (seanshpark) [inline compiler/luci/pass/include/luci/Pass/FuseBCQPass.h:34]: Doesn't seem to be for this PR

Comment (llFreetimell) [reply compiler/luci/pass/include/luci/Pass/FuseBCQPass.h:34]: This PR is for changing the `run(loco::Graph)` function to return false and it does not use the parameter.
As a result it caused compile error if I do not remove the unused parameter.

Comment (seanshpark) [reply compiler/luci/pass/include/luci/Pass/FuseBCQPass.h:34]: Really?

Comment (llFreetimell) [reply compiler/luci/pass/include/luci/Pass/FuseBCQPass.h:34]: Yes, that's why I added last commit.
(http://nnfw.mooo.com:8080/job/nnfw/job/master/job/pr-nncc-debug/4911/)

**Review by seanshpark — APPROVED** [review]
LGTM thank you

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**